PendingReleaseNotes

```diff
@@ -321,3 +321,7 @@
   by default. However, if enabled, user now have to pass the
   ``--yes-i-really-mean-it`` flag to ``osd pool set size 1``, if they are really
   sure of configuring pool size 1.
+
+* librbd clone now uses the default stripe_unit and stripe_count. This change
```

Comment:
Nit: the clone API always has used the default stripe unit/count. The change is to inherit the parent stripe width/count if not specified.